src/components/Hero/Hero.module.scss

```diff
@@ -50,6 +58,10 @@
     flex-direction: column;
     bottom: 2vh;
   }
+
+  @media (max-width: 281px) {
+    padding: 2vw;
```

Comment:
try to refrain from using vw/vh for anything other than the height/width of elements. best to just have consistent values for different layouts. I think I'm using 24 or 32 or mobile